sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeIdentityDocuments.java

```diff
@@ -40,18 +39,18 @@ public static void main(final String[] args) throws IOException {
             .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
             .buildClient();
 
-        File sourceFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/resources/java/"
+        File licenseDocumentFile = new File("../formrecognizer/azure-ai-formrecognizer/src/samples/resources/java/"
             + "sample-forms/ID documents/license.jpg");
```

Comment:
small thing, but might be worth renaming the folder where the sample files live to identity documents